client/dashboard/sites/settings-sftp-ssh/sftp-card.tsx

```diff
@@ -0,0 +1,119 @@
+	BaseControl,
+	Button,
+	Card,
+	CardBody,
+	ExternalLink,
+} from '@wordpress/components';
+import { useDispatch } from '@wordpress/data';
+import { createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { store as noticesStore } from '@wordpress/notices';
+import { siteSftpUsersResetPasswordMutation } from '../../app/queries';
+import ClipboardInputControl from './clipboard-input-control';
+import type { SftpUser } from '../../data/types';
+
+const SFTP_URL = 'sftp.wp.com';
+const SFTP_PORT = '22';
+
+export default function SftpCard( {
+	siteSlug,
+	sftpUsers = [],
+}: {
+	siteSlug: string;
+	sftpUsers: SftpUser[];
+} ) {
+	const { username, password } = sftpUsers[ 0 ] ?? {};
+	const mutation = useMutation( siteSftpUsersResetPasswordMutation( siteSlug ) );
+	const { createErrorNotice } = useDispatch( noticesStore );
+
+	const handleCreatePassword = () => {
+		mutation.mutate( username, {
+			onError: () => {
+				createErrorNotice(
+					__(
+						'Sorry, we had a problem retrieving your sftp user details. Please refresh the page and try again.'
+					),
+					{
+						type: 'snackbar',
+					}
+				);
+			},
+		} );
+	};
+
+	return (
+		<Card>
+			<CardBody>
+				<VStack style={ { paddingBottom: '12px' } }>
+					<Text size="15px" weight={ 500 } lineHeight="32px">
+						{ __( 'SFTP' ) }
+					</Text>
+					<Text variant="muted" as="p">
+						{ createInterpolateElement(
+							__(
+								'Use the credentials below to access and edit your website files using an SFTP client. <link>Learn more</link>.'
+							),
+							{
+								link: <ExternalLink href="#" children={ null } />,
+							}
+						) }
+					</Text>
+				</VStack>
+				<VStack spacing={ 4 } style={ { padding: '8px 0' } }>
+					<ClipboardInputControl
+						label={ __( 'URL' ) }
+						value={ SFTP_URL }
+						readOnly
+						__next40pxDefaultSize
+					/>
+					<ClipboardInputControl
+						label={ __( 'Port' ) }
+						value={ SFTP_PORT }
+						readOnly
+						__next40pxDefaultSize
+					/>
+					<ClipboardInputControl
+						label={ __( 'Username' ) }
+						value={ username }
+						readOnly
+						__next40pxDefaultSize
+					/>
+					{ password ? (
+						<ClipboardInputControl
+							label={ __( 'Password' ) }
+							value={ password }
+							help={ __(
+								'Save your password somewhere safe. You will need to reset it to view it again.'
+							) }
+							readOnly
+							__next40pxDefaultSize
+						/>
+					) : (
+						<BaseControl
+							label={ __( 'Password' ) }
+							help={ __( 'To maintain security, you must reset your password to view it.' ) }
+							__nextHasNoMarginBottom
+							children={ null }
+						/>
+					) }
+				</VStack>
+				{ ! password && (
+					<HStack style={ { padding: '8px 0' } }>
+						<Button
+							variant="secondary"
+							isBusy={ mutation.isPending }
+							onClick={ handleCreatePassword }
+						>
+							{ __( 'Reset password' ) }
+						</Button>
```

Comment:
Maybe it's just me but I *think* this should have `isDestructive` because it's resetting your existing password without any confirmation

Reply:
I think it’s fine because it’s not a scary button, and nothing will be broken or removed. People can feel free to reset their password if they forget it.

Reply:
dhW7zYBamXJIeJIE3wy5f4-fi-3074_108567!

Reply:
We have some guidelines around usage of `isDestructive` in the design system docs and it's mostly reserved for big actions like deleting an entire site. I don't think it's necessary here.

If we’re concerned that people will reset their password and break other apps that are relying on that password we should present a confirmation dialog instead.

![image](https://github.com/user-attachments/assets/94c56671-c625-44e6-91c0-52cb31e6414e)

Reply:
I see, thanks! Let's not change anything I think for now 😄